ballerina-tests/tests/01_listener_test.bal

```diff
@@ -26,3 +27,59 @@ function testInvalidMaxQueryDepth() returns error? {
     graphql:Error err = <graphql:Error>result;
     test:assertEquals(err.message(), "Max query depth value must be a positive integer");
 }
+
+graphql:Service greetingService = service object {
+    resource function get greeting() returns string {
+        return "Hello, World";
+    }
+};
+
+@test:Config {
+    groups: ["listener", "client"]
+}
+function testAttachingGraphQLServiceToDynamicListener() returns error? {
+    graphql:Listener serverEP = check new (9600);
```

Comment:
Let's use an existing listener in `test_listeners.bal` if possible.

Reply:
Updated